torchci/components/WorkflowBox.tsx

```diff
@@ -25,14 +27,28 @@ export default function WorkflowBox({
   workflowName: string;
   jobs: JobData[];
 }) {
+  const router = useRouter();
+  // This will scroll to the anchor when the page loads
+  const ref = useCallback(() => {
```

Comment:
We have a useScrollTo.ts hook that will take whatever's in the router and scroll to it.

Reply:
Ohhhh, that's good to know. I'm reinventing the wheel here without realizing it. TIL!

Reply:
This is so much cleaner.  Thank @zengk95!!